.github/workflows/e2e-component-tests-embedding-sdk.yml

```diff
@@ -129,6 +170,12 @@ jobs:
         with:
           name: metabase-ee-${{ github.event.pull_request.head.sha || github.sha }}-uberjar
 
+      - name: Download Embedding SDK artifact
```

Comment:
nit: I'd just move this down to replace the steps that you removed "Compile CLJS" and "Build Embedding SDK package" because they're equivalent. Even though doing this early doesn't change anything in the grand scheme of things, but I'd love to see stuff being consistent and true to the source. This would make reviewing diff easier in the long run.

Reply:
sounds good! lemme move it

Reply:
fixed in https://github.com/metabase/metabase/pull/55405/commits/6bcf6513b347d6f841b45d2b50f1b3a677ee77f7